tests/Php56/Php56Test.php

```diff
@@ -80,7 +80,7 @@ public function testLdapEscape($subject, $ignore, $flags, $expected)
     public function provideLdapEscapeValues()
     {
         return array(
-            array('foo=bar(baz)*', null, p::LDAP_ESCAPE_DN, 'foo\3dbar(baz)*'),
```

Comment:
Actually, the escape tests were backported from the tests from the official PHP interpreter.
See https://github.com/php/php-src/tree/master/ext/ldap/tests

For maintenance's sake, you should create a new test case line.

Cheers! :)